logging/config/cluster-logging-configuring-cr.adoc

```diff
@@ -1,11 +1,11 @@
 :context: cluster-logging-configuring-cr
 [id="cluster-logging-configuring-cr"]
-= About the Cluster Logging custom resource
+= About the Cluster Logging Custom Resource
```

Comment:
We use lowercase for "custom resource" so I think this was correct before. Seems it's  not called out explicitly in our guidelines, though I thought it was: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#object-references
(Same with "custom resource definition (CRD)", FYI)